wagtail/admin/templates/wagtailadmin/shared/chooser_breadcrumb.html

```diff
@@ -10,11 +10,9 @@
                     {% include "wagtailadmin/shared/locale_selector.html" with translations=locale_options locale=page.locale only %}
                 {% endif %}
             {% else %}
-                <div class="c-dropdown t-inverted {{ class }}" style="display: inline-block;" data-dropdown>
-                    <div aria-label="{{ page.locale.get_display_name }}" class="c-dropdown__button u-btn-current">
-                        {% icon name="site" class_name="default" %}
-                        {{ page.locale.get_display_name }}
-                    </div>
+                <div aria-label="{{ page.locale.get_display_name }}" class="breadcrumb-link">
```

Comment:
annoyingly this results in white on white if you start drilling down (e.g. home > blog)
the stylesheet caters for `.page-editor .modal .breadcrumb a` (giving it the body colour)

Reply:
Seems fine here (tried rebasing on current main in case something in the new designs has broken it) - am I missing something in the setup?
![Screenshot 2022-04-11 at 13 23 47](https://user-images.githubusercontent.com/85097/162738622-05618586-1e42-47de-a326-b98ae6ebb92e.png)